### ISSUE TITLE
Add more filters to the query log

### ISSUE THREAD

**Issue by jimbo54321** [AdguardTeam/AdGuardHome#882]
It would be good if you could view the URL of what adult sites were blocked as apposed to just seeing the number of adult sites blocked.

Thanks

**Comment by ameshkov** [comment]
I guess we should add more filters here:
https://uploads.adguard.com/up04_uen8h_AdGuard_Home.png

For instance:
1. Show blocked by filters
2. Show blocked adult websites
3. Show blocked dangerous websites
4. Show whitelisted

**Comment by ArchiveBase** [comment]
Source https://github.com/AdguardTeam/AdGuardHome/issues/882 
> @Onepamopa
would be nice if we can get graphs & statistics (total Nr of DNS requests, blocked by filters, malware/phishing, adult

**Comment by ameshkov** [comment]
Done in v0.103

**Comment by Redundanz** [comment]
> I guess we should add more filters here: https://uploads.adguard.com/up04_uen8h_AdGuard_Home.png
> 
> For instance:
> 
> 1. Show blocked by filters

Hi!
I didn't want to open a new issue right away since Github advised me to search for other issues with the same topic. Found some relevant entries, while this one seems the most fitting to me as it addresses the exact same enhancement that I am missing as of now.

So the query log tends to be full of different stuff as is expected. But when there are lots of consecutive service blocks, for instance, then it is really hard to quickly check for the filter rule based blocking without sometimes scrolling pages and pages through the query log. The list of possible query log filters is no doubt long, yet it is missing the most important one (in my eyes) 
-> "Blocked by (custom) filters / filter lists" which was also enumerated here for implementation, as seen in comment further above.

It would be really great and helpful to have this if there is no technical or purposeful reason to leave it out!

thkx
Red
